CONTRIBUTING.md

```diff
@@ -245,16 +258,38 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something.
 
+Notes: if the reviewer mentions *nits* in their comments, that means
+"requests for changes that are not essential". Usually these nits are
+related to coding styles or typos. It's always a good idea to pay
+attention to the detail!
 
 ### Step 8: Landing
 
-Once your Pull Request has been reviewed and approved by at least one Node.js
-Collaborators (often by saying LGTM, or Looks Good To Me), and as long as
-there is consensus (no objections from a Collaborator), a
-Collaborator can merge the Pull Request . GitHub often shows the Pull Request as
- `Closed` at this point, but don't worry. If you look at the branch you raised
- your Pull Request against (probably `master`), you should see a commit with
- your name on it. Congratulations and thanks for your contribution!
+A Pull Request needs to be reviewed and approved by at least one Node.js
+Collaborators (often by saying LGTM, or Looks Good To Me) to get landed.
```

Comment:
Do you mean this should be singular? I will add a phrase mentioning the Approve button, thanks